tests/integration/modules/molecule/role/Dockerfile.j2

```diff
@@ -0,0 +1,14 @@
+# Molecule managed
```

Comment:
This should be a symlink pointing to `../shared/Dockerfile.j2`

Reply:
Oops, I totally missed this. Done, thanks :+1: